### PR TITLE
feat: add k8s cis bench

### LINKED ISSUE #3276: Initial CIS compliance
with Infra checks, and compliance framework, we have a good base to add initial CIS compliance report in Trivy. It will be missing node checks (e.g file permissions) but 1) for managed cluster, it's less relevant 2) we can add node checks in a later stage

### PR DESCRIPTION
**Pull request by chen-keinan**
Signed-off-by: chenk <hen.keinan@gmail.com>

## Related issues
- Close #3276

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).

### CODE REVIEW

File: docs/docs/kubernetes/cli/cis.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+```
+$ trivy k8s cluster --compliance=cis --report summary --format json
+```
+
+```
+$ trivy k8s cluster --compliance=cis --report all --format json
+```
+
+## Custom compliance report
```
Comment (knqyf263) [inline docs/docs/kubernetes/cli/cis.md:114]: Seems like the custom report is also described in the NSA page. We may want to add `Overview` under `Kubernetes/CLI/Compliance` and explain that Trivy supports NSA and CIS Benchmark both. Also, we can mention custom reports in that page.

File: mkdocs.yml [hunk 2]
```diff
@@ -82,7 +82,9 @@ nav:
       - Kubernetes:
           - CLI:
               - Scanning: docs/kubernetes/cli/scanning.md
-              - Compliance: docs/kubernetes/cli/compliance.md
+              - Compliance: 
+                - NSA: docs/kubernetes/cli/nsa.md
```
Comment (knqyf263) [inline mkdocs.yml:86]: I want to get the directory structure aligned with the document structure.

```suggestion
                - NSA: docs/kubernetes/cli/compliance/nsa.md
```

File: docs/docs/kubernetes/cli/cis.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+The Trivy K8s CLI allows you to create a custom compliance specification and pass it to trivy for generating scan report .
+
+The report is generated based on scanning result mapping between users define controls and trivy checks ID.
+The supported checks are from two types and can be found at [Aqua vulnerability DB](https://avd.aquasec.com/):
+- [misconfiguration](https://avd.aquasec.com/misconfig/)
+- [vulnerabilities](https://avd.aquasec.com/nvd)
```
Comment (knqyf263) [inline docs/docs/kubernetes/cli/cis.md:121]: It is a bit tricky, but a new line is needed for bullet points in mkdocs.

```suggestion
The supported checks are from two types and can be found at [Aqua vulnerability DB](https://avd.aquasec.com/):

- [misconfiguration](https://avd.aquasec.com/misconfig/)
- [vulnerabilities](https://avd.aquasec.com/nvd) 
```

You can build the doc by `make mkdocs-serve`.

image

**Review by knqyf263 — COMMENTED** [review]
nit: Can we left-align `Control Name` in the table?

**Review by knqyf263 — APPROVED** [review]
We may want to improve the document a bit more, but we can do that in another PR.
https://github.com/aquasecurity/trivy/pull/3340

File: go.mod [hunk 4]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alicebob/miniredis/v2 v2.23.0
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/defsec v0.82.6
+	github.com/aquasecurity/defsec v0.82.7-0.20221225070347-3a6cfb67e460
```
Comment (simar7) [inline go.mod:11]: @chen-keinan - this PR introduced a version of defsec that isn't released, I'm curious what version is this? The released are here https://github.com/aquasecurity/defsec/releases

**Pull request merged**